### ISSUE TITLE
BuildingsPy Exception: can only concatenate tuple (not "str") to tuple.

### ISSUE THREAD

**Issue by Shalom6624** [lbl-srg/BuildingsPy#574]
Hello all,

Please I am trying use the Buildingspy to run modelica simulations from python. I have installed all necessary modules. But on executing my code, I received the error:

Exception: can only concatenate tuple (not "str") to tuple.
   You need to delete the directory manually.

To try to debug the problem, I then tested with an example model from [example code](https://simulationresearch.lbl.gov/modelica/buildingspy/examples.html) (see code below)

```#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
from multiprocessing import Pool

from buildingspy.simulate.Dymola import Simulator


# Function to set common parameters and to run the simulation
def simulateCase(s):
    """ Set common parameters and run a simulation.

    :param s: A simulator object.

    """
    s.setStopTime(24 * 3600)
    # Kill the process if it does not finish in 1 minute
    s.setTimeOut(60)
    s.printModelAndTime()
    s.simulate()


def main():
    """ Main method that configures and runs all simulations
    """
    import shutil

    # Build list of cases to run
    li = []
    # First model, from Modelica Buildings Library, v9.1.0
    model = 'Buildings.Fluid.SolarCollectors.Examples.FlatPlateWithTank'
    s = Simulator(model, outputDirectory='case1')
    s.addParameters({'tan.VTan': 1.5})
    li.append(s)
    # second model
    s = Simulator(model, outputDirectory='case2')
    s.addParameters({'tan.VTan': 2})
    li.append(s)

    # Run all cases in parallel
    po = Pool()
    po.map(simulateCase, li)
    po.close()
    po.join()

    # Clean up
    shutil.rmtree('case1')
    shutil.rmtree('case2')


# Main function
if __name__ == '__main__':
    main()
```

But got similar error as shown below
*** Error: Simulation failed in 'case2'
   Exception: can only concatenate tuple (not "str") to tuple.
   You need to delete the directory manually.

Please any help on how to resolve this?

**Comment by mwetter** [comment]
I cannot reproduce the exception
```
Exception: can only concatenate tuple (not "str") to tuple.
```
However, if I simulate the model from Buildings v9.1.0, I receive an error
```
*** Error: Log file contained the line ' = false'. Preceeding line: 'simulateModel(modelInstance, startTime=0, stopTime=86400, method="radau", tolerance=1e-06, resultFile="FlatPlateWithTank", numberOfIntervals=500);
```
You can either use either release 9.1.2, or release 11. Both correct an error in the unit specification that recent Dymola versions catch and cause it to stop with an error.

Also, make sure you set the `MODELICAPATH` environment variable.

If problems persist, you can try to manually run 
```
cd case2
dymola run.mos
```
ideally after you delete the last line that has the `exit` command.

**Comment by Shalom6624** [comment]
Dear Michael @mwetter, thanks for your feedback. I have tried with buildings 11.0.0 and 10.0.0 but still get the same error. Also, I have my MODELICAPATH in my system environment variables configured to include the path to my Dymola executable (C:\Program Files\Dymola 2024x\bin64). Also, I tried out your suggestion to run "dymola run.mos" manually and it opened dymola but with an error that reads **"Did not find model Buildings.Fluid.SolarCollectors.Examples.FlatPlateWithTank(tan.VTan=2)"**. I noticed that the code starts dymola it does not load the library. Is this a problem from my end? Also, do I need to install the buildings library to a specific folder? Or what might be the omission at my end?
